CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.4 (2018-11-30)
+
+- Fixed `SkyListViewGridsModule` exports to include required submodules and `SkyGridModule`. [#9](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/9)
```

Comment:
Based on https://github.com/blackbaud/skyux-list-builder-view-grids/pull/9/files#diff-936142aada680d552b8aa57cc92b0e3e, we have a stray "s" in the module name. It should be "SkyListViewGridModule"

Reply:
Ah yeah, nice catch. Fixed